libs/api/domains/health-insurance-v2/src/lib/graphql/health-insurance-v2.resolver.ts

```diff
@@ -0,0 +1,39 @@
+import { Inject, UseGuards } from '@nestjs/common'
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { Audit } from '@island.is/nest/audit'
+import { IdsUserGuard, ScopesGuard } from '@island.is/auth-nest-tools'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import type { Logger } from '@island.is/logging'
+import { AccidentStatusInput } from './dto/AccidentStatus.input'
+import { AccidentNotificationStatus } from './models/accidentNotificationStatus.model'
+import { HealthInsuranceService } from './health-insurance-v2.service'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Resolver(() => AccidentNotificationStatus)
+export class HealthInsuranceResolver {
+  constructor(
+    private accidentNotificationApi: HealthInsuranceService,
+    @Inject(LOGGER_PROVIDER)
+    private logger: Logger,
+  ) { }
+
+  @Query(() => AccidentNotificationStatus, {
+    name: 'accidentStatus',
```

Comment:
Should follow the [graphql naming convention](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-naming-conventions-into-shared-api).